functions.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+function checkSingleCardAndGetAllInfo(string $cardTitle): array {
+    $connectionString = 'mysql:host=db; dbname=mtg_cards';
+    $dbUsername = 'root';
+    $dbPassword = 'password';
+    $db = new PDO ($connectionString, $dbUsername, $dbPassword);
```

Comment:
should be without the space after PDO: `new PDO(....)`

Reply:
Deleted the function but have fixed in the other function that checks the db for the display cards.